flower_shop/views/flower_only_products_menu.xml

```diff
@@ -0,0 +1,9 @@
+<odoo>
+    <data>
+        <record model="ir.ui.menu" id="menu_flower_only_products">
+            <field name="name">Flower Products</field>
+            <field name="sequence" eval="10"/>
+            <field name="action" ref="action_flower_only_products"/>
```

Comment:
I understand if you're experimenting, but It's preferable to use the **menuitem** tag when you want to define a menu item, for the sake of readability and scaling.